php/tower/config/server.php

```diff
@@ -4,7 +4,7 @@
     'listen' => 'http://0.0.0.0:3000',
     'transport' => 'tcp',
     'context' => [],
-    'count' => cpuCoreCount() * 2,
+    'count' => 8,

```

Comment:
why fix the CPU core num ? I mean that in the near futur,  we are going to upgrade infrastructure were benchmark runs (and then computer could have more than 8 core)

Reply:
hi @waghanza 

okay thanks
 
So please merge my new commit